files/es/web/guide/ajax/index.md

```diff
@@ -56,8 +56,14 @@ Una introducción a AJAX.
 - [Kit de Integración Flash/AJAX](http://www.osflash.org/doku.php?id=flashjs)
 - [Una Simple Librería de Interfaz XMLHTTP](http://xkr.us/code/javascript/XHConn/)
 
-<!---->
-
-- #### Temas relacionados
-
-  - : [HTML](es/HTML) · [XHTML](es/XHTML) · [CSS](es/CSS) · [DOM](es/DOM) · [JavaScript](es/JavaScript) · [XML](es/XML) · [XMLHttpRequest](es/XMLHttpRequest) · [XSLT](es/XSLT) · [DHTML](es/DHTML)
+#### Temas relacionados
+
+- [HTML](es/HTML)
```

Comment:
```suggestion
- [HTML](/es/HTML)
```

Reply:
You're right, I think I cleanup up a bunch of those in another PR. I'll do a quick scan to see if there are any more in this folder and push the changes if there are a bunch

Reply:
Took a pass through and cleaned up a few otheres I found in the ES files